test/powershell/Modules/Microsoft.PowerShell.Utility/Sort-Object.Tests.ps1

```diff
@@ -21,204 +21,946 @@
 }
 
 Describe "Sort-Object DRT Unit Tests" -Tags "CI" {
-	It "Sort-Object with object array should work"{
```

Comment:
As per our guidelines, avoid formatting changes. I realize test files are inconsistent w.r.t. tabs vs. spaces, but we want to keep meaningful additions in different commits from formatting changes.